<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="callName">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="callButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Call</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopCall">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Hang Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerTxt">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="partnerGraphicsView">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>takeScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>callButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startCall()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startCall()</slot>
 </slots>
</ui>
